app/models/podcast_episode.rb

```diff
@@ -15,7 +15,7 @@ class PodcastEpisode < ApplicationRecord
   delegate :published, to: :podcast
 
   belongs_to :podcast
-  has_many :comments, as: :commentable, inverse_of: :commentable
+  has_many :comments, as: :commentable, inverse_of: :commentable, dependent: :nullify
```

Comment:
Removing a podcast episode should orphan the comments, not remove them. Comments have unique URLs that can survive the destruction of the object they are attached to